package.json

```diff
@@ -65,7 +65,7 @@
     "webmaker-translation-stats": "0.0.1"
   },
   "scripts": {
-    "postinstall": "node_modules/.bin/bower install && node_modules/.bin/grunt requirejs:dist",
+    "postinstall": "node_modules/.bin/bower install && grunt requirejs:dist",
```

Comment:
This is what I wondered, thank you for looking into it, and thank you for this patch.  Nice work.